.github/workflows/main.yml

```diff
@@ -0,0 +1,51 @@
+name: tt Run and Publish
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [main]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch: 
+    inputs:
+      glossaryDefFile:
+        required: true
+        description: glossaryDefFile
+      deployBranch:
+        required: false
+        description: This is where the output of script will be pushed to.
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
This checkout action checks out the current repo -- right?

That's not quite what we need. Maybe it would be easier to clarify on a call...